Sources/DatadogSDKTesting/Utils/NTPClock.swift

```diff
@@ -0,0 +1,19 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2020-2021 Datadog, Inc.
+ */
+
+import Foundation
+@_implementationOnly import Kronos
+@_implementationOnly import OpenTelemetrySdk
+
+class NTPClock: OpenTelemetrySdk.Clock {
+    init() {
+        Kronos.Clock.sync()
+    }
+
+    var now: Date {
+        return Kronos.Clock.now ?? Date()
+    }
```

Comment:
Mind this trap: https://github.com/DataDog/dd-sdk-ios/issues/588 - `Kronos` requires that both `Clock.sync()` and `Clock.now` must be called on the main thread (@maxep gave more context here: https://github.com/MobileNativeFoundation/Kronos/issues/85#issuecomment-925037402). Is this ensured by design in `dd-swift-testing`? If not, I think this could be failing if TSAN is enabled in tests.